app/controllers/projects/accounts/settings_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Projects::Accounts::SettingsController < ApplicationController
+  skip_after_action :verify_authorized, :verify_policy_scoped
+
+  def show
+    @project_role = ProjectRole.find_by(project_id: params[:project_id], account_id: params[:account_id])
```

Comment:
```suggestion
    @project_role = ProjectRole.find_by!(project_id: params[:project_id], account_id: params[:account_id])
```

Otherwise it may raise 500 on next line.